generators/languages/templates/src/main/webapp/i18n/by/global.json.ejs

```diff
@@ -0,0 +1,160 @@
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+{
+    "global": {
+        "title": "<%= capitalizedBaseName %>",
+        "browsehappy": "Dаш браўзэр <strong>састарэў</strong>. Калі ласка <a href=\"http://browsehappy
+        .com/?locale=ru\">абновіце ваш браўзэр</a>.",
+        "menu": {
+            "home": "Галоўная",
+            "jhipster-needle-menu-add-element": "JHipster will add additional menu entries here (do not translate!)",
+            "entities": {
+                "main": "Cутнасці",
+                "jhipster-needle-menu-add-entry": "JHipster will add additional entities here (do not translate!)"
+            },
+            "account": {
+                "main": "Профіль",
+                "settings": "Налады",
+                "password": "Пароль",
+                "sessions": "Сесіі",
+                "login": "Увайсці",
+                "logout": "Выйсці",
+                "register": "Зарэгістравацца"
+            },
+            "admin": {
+                "main": "Адміністраванне",
+                <%_ if (applicationType === 'gateway') { _%>
+                "gateway": "Шлюз",
+                <%_ } _%>
+                "userManagement": "Кіраванне карыстальнікамі",
+                "tracker": "Трэкер карыст.",
+                "metrics": "Метрыкі",
+                "health": "Стан",
+                "configuration": "Налады",
+                "logs": "Логі",
+                <%_ if (databaseType !== 'no' && databaseType !== 'cassandra') { _%>
+                "audits": "Аўдыт",
+                <%_ } _%>
+                "apidocs": "API",
+                "database": "База дадзеных",
+                "jhipster-needle-menu-add-admin-element": "JHipster will add additional menu entries here (do not translate!)"
+            },
+            "language": "Мова"
+        },
+        "form": {
+            "username": "Лагін",
+            "username.placeholder": "Ваш лагін",
+            "currentpassword": "Цяперашні пароль",
+            "currentpassword.placeholder": "Цяперашнія пароль",
+            "newpassword": "Новы пароль",
+            "newpassword.placeholder": "Новы пароль",
+            "confirmpassword": "Пацвярджэнне новага паролю",
+            "confirmpassword.placeholder": "Пацвердзіце новы пароль",
+            "email": "Email",
+            "email.placeholder": "Ваш email"
+        },
+        "messages": {
+            "info": {
+                "authenticated": {
+                    "prefix": "Калі вы хочаце ",
+                    "link": "аўтарызавацца",
+                    "suffix": ", вы можаце паспрабаваць акаўнты па змаўчанні:<br/>- Адміністратар (лагін=\"admin\" і
+                    пароль=\"admin\") <br/>- Карыстальнік (лагін=\"user\" і пароль=\"user\")."
```

Comment:
same comment: it should not be a carriage return here

Reply:
Done